packages/@react-types/shared/src/collections.d.ts

```diff
@@ -96,28 +96,28 @@ export type SortDirection = 'ascending' | 'descending';
 
 export interface KeyboardDelegate {
   /** Returns the key visually below the given one, or `null` for none. */
-  getKeyBelow?(key: Key): Key | null,
+  getKeyBelow?(key: Key | null): Key | null,
```

Comment:
Is it actually valid to call `getKeyBelow` with `null`? What should be the result in that case? Should this instead be checked at the call site?